usb-serial: ti_usb, TI ez430 development tool ID

Cc: Greg KH <greg@kroah.com>
Signed-off-by: Oleg Verych <olecom@flower.upol.cz>
---
  Please apply to stable and rc kernels. Thanks.


 drivers/usb/serial/ti_usb_3410_5052.c |    2 ++
 drivers/usb/serial/ti_usb_3410_5052.h |    1 +
 2 files changed, 3 insertions(+)

Index: linux-source-2.6.18/drivers/usb/serial/ti_usb_3410_5052.c
===================================================================
*** linux-source-2.6.18.orig/drivers/usb/serial/ti_usb_3410_5052.c	2006-11-17 08:54:28.378314000 +0100
--- linux-source-2.6.18/drivers/usb/serial/ti_usb_3410_5052.c	2006-11-17 07:10:45.000000000 +0100
***************
*** 228,233 ****
--- 228,234 ----
  /* null entry */
  static struct usb_device_id ti_id_table_3410[1+TI_EXTRA_VID_PID_COUNT+1] = {
  	{ USB_DEVICE(TI_VENDOR_ID, TI_3410_PRODUCT_ID) },
+         { USB_DEVICE(TI_VENDOR_ID, TI_3410_EZ430_ID) },
  };
  
  static struct usb_device_id ti_id_table_5052[4+TI_EXTRA_VID_PID_COUNT+1] = {
***************
*** 239,244 ****
--- 240,246 ----
  
  static struct usb_device_id ti_id_table_combined[] = {
  	{ USB_DEVICE(TI_VENDOR_ID, TI_3410_PRODUCT_ID) },
+         { USB_DEVICE(TI_VENDOR_ID, TI_3410_EZ430_ID) },
  	{ USB_DEVICE(TI_VENDOR_ID, TI_5052_BOOT_PRODUCT_ID) },
  	{ USB_DEVICE(TI_VENDOR_ID, TI_5152_BOOT_PRODUCT_ID) },
  	{ USB_DEVICE(TI_VENDOR_ID, TI_5052_EEPROM_PRODUCT_ID) },
Index: linux-source-2.6.18/drivers/usb/serial/ti_usb_3410_5052.h
===================================================================
*** linux-source-2.6.18.orig/drivers/usb/serial/ti_usb_3410_5052.h	2006-11-17 08:54:25.402128000 +0100
--- linux-source-2.6.18/drivers/usb/serial/ti_usb_3410_5052.h	2006-11-17 08:49:40.000000000 +0100
***************
*** 28,33 ****
--- 28,34 ----
  /* Vendor and product ids */
  #define TI_VENDOR_ID			0x0451
  #define TI_3410_PRODUCT_ID		0x3410
+ #define TI_3410_EZ430_ID		0xF430  /* TI ez430 development tool */
  #define TI_5052_BOOT_PRODUCT_ID		0x5052	/* no EEPROM, no firmware */
  #define TI_5152_BOOT_PRODUCT_ID		0x5152	/* no EEPROM, no firmware */
  #define TI_5052_EEPROM_PRODUCT_ID	0x505A	/* EEPROM, no firmware */


-- 
To UNSUBSCRIBE, email to debian-kernel-REQUEST@lists.debian.org
with a subject of "unsubscribe". Trouble? Contact listmaster@lists.debian.org

---------------------------------------------------------------------------------------
Orange vous informe que cet  e-mail a ete controle par l'anti-virus mail. 
Aucun virus connu a ce jour par nos services n'a ete detecte.



